implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -779,6 +779,10 @@ impl NodeManagerWorker {
                 self.create_project_response(ctx, dec.decode()?, space_id)
                     .await?
             }
+            (Get, ["version_info"]) => {
```

Comment:
I'd keep the ["v0", "projects", "version_info"] route. Note that in the rust context, we are managing a node (the NodeManager) that has its own version information, so it would be nice to have this method under the "projects" scope to make more explicit what version are we talking about.

Reply:
I think it's nicer too. Added in latest commit.